test/testcafe/framework/page-objects/components/BaseFormObject.js

```diff
@@ -337,6 +337,19 @@ export default class BaseFormObject {
   }
 
   async selectRadioButtonOptionByValue(fieldName, value) {
+    if (userVariables.v3) {
+      const radioOption = await this.el.find(`input[type="radio"][value="${value}"]`);
+      await this.t.click(radioOption); 
+
```

Comment:
Getting the label text associated with a radio button is not very clean.
This is how it looks in the DOM - 
<img width="533" alt="Screen Shot 2022-12-01 at 3 06 04 PM" src="https://user-images.githubusercontent.com/27521424/205179179-d7f90910-44e3-409e-9b6a-bfea6f386134.png">

Getting parent of parent works but is there a better way to do this?